pkgs/tools/misc/otel-cli/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildGoModule, fetchFromGitHub, getent }:
+
+buildGoModule rec {
+  pname = "otel-cli";
+  version = "0.0.20";
+
+  src = fetchFromGitHub {
+    owner = "equinix-labs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-bWdkuw0uEE75l9YCo2Dq1NpWXuMH61RQ6p7m65P1QCE=";
+  };
+
+  vendorHash = "sha256-IJ2Gq5z1oNvcpWPh+BMs46VZMN1lHyE+M7kUinTSRr8=";
+
+  nativeBuildInputs = [ getent ];
+
+  preCheck = ''
+    ln -s $GOPATH/bin/otel-cli .
+    substituteInPlace main_test.go --replace \
+      'const minimumPath = `/bin:/usr/bin`' \
+      'const minimumPath = `'"$PATH"'`'
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/equinix-labs/otel-cli";
+    description = "a cli tool for otel instrumenting command line work";
```

Comment:
https://nixos.org/manual/nixpkgs/stable/#var-meta-description says "Capitalise the first character."

I think a clearer description would be "A command-line tool for sending OpenTelemetry traces" (from the beginning of the project README).

Reply:
This doesn't appear to have changed.

Reply:
sorry, I had a number of changes I was wrangling today, this one got skipped, I will make sure to do this before resolving it again.